### ISSUE TITLE
Ending a Tween

### ISSUE THREAD

**Issue by ADuckInBlack** [ShadowMario/FNF-PsychEngine#9139]
### What is your question?

First, I wanna say I'm fairly new to coding on psych engine (and coding in general), so my code's a bit messy.
I wanna ask, does anybody know how to stop a tween and put the arrows back where they belong?
I'd appreciate the help!
Judging my code is allowed.
Yes, I did base my code off a template.
![image](https://user-images.githubusercontent.com/33814766/174512739-a09ce7ea-b2bd-4a49-89ab-8401676769f9.png)

**Comment by bruh1cat** [comment]
function onTweenCompleted()
if tag == 'tag' then
blah blah blah
end
end

This?

> ### What is your question?
> First, I wanna say I'm fairly new to coding on psych engine (and coding in general), so my code's a bit messy. I wanna ask, does anybody know how to stop a tween and put the arrows back where they belong? I'd appreciate the help! Yes, I did base my code off a template. (And by that I mean I stole this from the Psych wiki) ![image](https://user-images.githubusercontent.com/33814766/174512739-a09ce7ea-b2bd-4a49-89ab-8401676769f9.png)

**Comment by ADuckInBlack** [comment]
> function onTweenCompleted() if tag == 'tag' then blah blah blah end end
> 
> This?
> 
> > ### What is your question?
> > First, I wanna say I'm fairly new to coding on psych engine (and coding in general), so my code's a bit messy. I wanna ask, does anybody know how to stop a tween and put the arrows back where they belong? I'd appreciate the help! Yes, I did base my code off a template. (And by that I mean I stole this from the Psych wiki) ![image](https://user-images.githubusercontent.com/33814766/174512739-a09ce7ea-b2bd-4a49-89ab-8401676769f9.png)

I'll give it a shot

**Comment by ADuckInBlack** [comment]
> function onTweenCompleted() if tag == 'tag' then blah blah blah end end
> 
> This?
> 
> > ### What is your question?
> > First, I wanna say I'm fairly new to coding on psych engine (and coding in general), so my code's a bit messy. I wanna ask, does anybody know how to stop a tween and put the arrows back where they belong? I'd appreciate the help! Yes, I did base my code off a template. (And by that I mean I stole this from the Psych wiki) ![image](https://user-images.githubusercontent.com/33814766/174512739-a09ce7ea-b2bd-4a49-89ab-8401676769f9.png)

I don't think this is _quite_ what i'm looking for. At the bottom of the image there's a line of code that puts the arrows back ALMOST where they're usually supposed to be, but it's not perfect. It's also a lot of code. If you could tell me how to put them back where they're supposed to be, that'd be great! If not, I could probably find a way to find where the arrows are on the X Axis to perfect the arrow's positions.

**Comment by bruh1cat** [comment]
Is it all the arrows that are almost where they were or is it one?

**Comment by ADuckInBlack** [comment]
> Is it all the arrows that are almost where they were or is it one?

All of them, but it was a random guess and I don't like how they're just ALMOST there. I wanna be more precise.

**Comment by bruh1cat** [comment]
What version are you using bc when I replicated the script only one note didn’t move to their original place.

**Comment by ADuckInBlack** [comment]
> What version pf psych are you using bc when I replicated the script only one note didn’t move to their original place.

Psych Engine 0.5.1
I also forgot to mention that the code is for an event

**Comment by bruh1cat** [comment]
> > What version pf psych are you using bc when I replicated the script only one note didn’t move to their original place.
> 
> Psych Engine 0.5.1 I also forgot to mention that the code is for an event

Maybe that's why mine almost worked bc I'm on a different version of psych (0.52h)

**Comment by bruh1cat** [comment]
> > What version pf psych are you using bc when I replicated the script only one note didn’t move to their original place.
> 
> Psych Engine 0.5.1 I also forgot to mention that the code is for an event

It's the same except that it's triggered by an event

**Comment by ADuckInBlack** [comment]
Thanks for the help, guys!
